pkgs/dart_pad/lib/elements/analysis_results_controller.dart

```diff
@@ -157,42 +150,42 @@ class AnalysisResultsController {
-          // line number and charStart could have been adjust because of an
-          // appended test, and use the information for the parentIssue instead.
-          // (It would probably be safe to always do this, but by doing this
-          // we DO NOT change any behavior except for when we have changed the
-          // sourceName to `test.dart` (the only way the sourceName can currently
-          // change until multi file source merged))
-          //TODO: @timmaffett For now we assume only 2 possibilities, 'main.dart'
-          // or 'test.dart' (and in that case we changed line# and charStart).
-          line: parentIssue.sourceName == 'main.dart'
-              ? diagnosticMessage.line
-              : parentIssue.line,
-          charStart: parentIssue.sourceName == 'main.dart'
-              ? diagnosticMessage.charStart
-              : parentIssue.charStart,
-          charLength: parentIssue.sourceName == 'main.dart'
-              ? diagnosticMessage.charLength
-              : parentIssue.charLength));
-    });
-
-    return elem;
-  }
+  // Element _createDiagnosticElement(
+  //     DiagnosticMessage diagnosticMessage, AnalysisIssue parentIssue) {
+  //   final message = diagnosticMessage.message;
+
+  //   final elem = DivElement()..classes.addAll(['message', 'clickable']);
+  //   elem.text = message;
+  //   elem.onClick.listen((event) {
+  //     // Stop the mouse event so the outer issue mouse handler doesn't process
+  //     // it.
+  //     event.stopPropagation();
+
+  //     _onClickController.add(Location(
+  //         //TODO: @timmaffett multi files will need -> diagnosticMessage.sourceName,
+  //         sourceName: parentIssue.sourceName,
+  //         // For now if the source name is NOT main.dart then ASSUME that the
+  //         // line number and charStart could have been adjust because of an
+  //         // appended test, and use the information for the parentIssue instead.
+  //         // (It would probably be safe to always do this, but by doing this
+  //         // we DO NOT change any behavior except for when we have changed the
+  //         // sourceName to `test.dart` (the only way the sourceName can currently
+  //         // change until multi file source merged))
+  //         //TODO: @timmaffett For now we assume only 2 possibilities, 'main.dart'
+  //         // or 'test.dart' (and in that case we changed line# and charStart).
+  //         line: parentIssue.sourceName == 'main.dart'
+  //             ? diagnosticMessage.line
+  //             : parentIssue.line,
+  //         charStart: parentIssue.sourceName == 'main.dart'
+  //             ? diagnosticMessage.charStart
+  //             : parentIssue.charStart,
+  //         charLength: parentIssue.sourceName == 'main.dart'
+  //             ? diagnosticMessage.charLength
+  //             : parentIssue.charLength));
+  //   });
+
+  //   return elem;
+  // }
```

Comment:
Why is this commented out?

Reply:
We no longer pass back an analysis issues context messages (here, called diagnostics). These are optional metadata about an issue. The user has enough to go on with the analysis issue type, message, and location. (so, I removed this UI as we no longer have the model information that is was using)